.vsts-dotnet-ci.yml

```diff
@@ -70,7 +70,7 @@ jobs:
     pool:
       name: NetCore-Public
       demands: ImageOverride -equals build.Ubuntu.2204.amd64.open
-    helixQueue: Ubuntu.2204.ArmArch.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-18.04-helix-arm32v7
+    helixQueue: Ubuntu.2204.ArmArch.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:debian-12-helix-arm32v7
```

Comment:
Why not use [ubuntu-24.04-helix-arm32v7](https://github.com/dotnet/dotnet-buildtools-prereqs-docker/blob/fa7c4c7e6c6f382defff193f310824e3dbce876b/src/ubuntu/manifest.json#L286)

Reply:
I was told specifically to use this one by @jakobbotsch because all Ubuntu containers were broken in helix.

Reply:
And now they may be fixed with https://github.com/dotnet/dotnet-buildtools-prereqs-docker/pull/1374 so I can revert back.